src/patternfly/components/Select/select.scss

```diff
@@ -22,9 +22,11 @@
   --pf-c-select__toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
   --pf-c-select__toggle--expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
   --pf-c-select__toggle--expanded--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-select__toggle--disabled--BackgroundColor: var(--pf-global--disabled-color--300);
   --pf-c-select__toggle--m-plain--BorderColor: transparent;
   --pf-c-select__toggle--m-plain--Color: var(--pf-global--Color--200);
   --pf-c-select__toggle--m-plain--hover--Color: var(--pf-global--Color--100);
+  --pf-c-select__toggle--m-plain--disabled--Color: var(--pf-global--disabled-color--200);
```

Comment:
This var is unused. I'll comment in the issue and see if we even want to support a plain variation of the select component.

Reply:
Hey @boaz0 we do want to support this variation, but you can remove this var. The disabled plain select should display the same as the regular disabled select.